rules/aws/initial_access_root_console_failure_brute_force.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/07/21"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of failed authentication attempts to the AWS management console for the Root user identity. An
+adversary may attempt to brute force the password for the Root user identity, as it has complete access to all services
+and resources for the AWS account.
+"""
+from = "now-20m"
```

Comment:
Can you add some FP language similar to the  `Okta Brute Force or Password Spraying Attack` rule:

_Automated processes that attempt to authenticate using expired credentials and unbounded retries may lead to false positives._